buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -410,6 +410,7 @@ class BeamModulePlugin implements Plugin<Project> {
         netty_tcnative_boringssl_static             : "io.netty:netty-tcnative-boringssl-static:2.0.8.Final",
         netty_transport_native_epoll                : "io.netty:netty-transport-native-epoll:$netty_version",
         postgres                                    : "org.postgresql:postgresql:$postgres_version",
+        powermock                                   : "org.powermock:powermock-mockito-release-full:1.6.4",
```

Comment:
Please add this new package into the ownership/JAVA_DEPENDENCY_OWNERS.yaml. You can leave the 'owners' blank if you don't want to take the ownership.